### PR TITLE
Bz1635743

### PR DESCRIPTION
**Pull request by jhoyt-rh**
OED Bug BZ1635743 (appending items to OPTIONS_ line)

@openshift/team-documentation, PTAL

@yinzhou, please review

@gkimetto, please review

### CODE REVIEW

File: install/host_preparation.adoc [hunk 1]
```diff
@@ -774,15 +774,16 @@ of log files.
...
 |===
 
-For example, to set the maximum file size to 1MB and always keep the last three
-log files, edit the *_/etc/sysconfig/docker_* file to configure `max-size=1M`
-and `max-file=3`:
-
+. To configure the log file, edit the *_/etc/sysconfig/docker_* file.
+For example, to set the maximum file size to 1MB and always retain the last three
+log files, append `max-size=1M` and `max-file=3` to the `OPTIONS=` line,
+being sure to keep them within the single quotes:
```
Comment (ahardin-rh) [inline install/host_preparation.adoc]: s/being sure to keep them within the single quotes/ensuring that the values maintain the single quotation mark formatting

**Comment by ahardin-rh** [comment]
@jhoyt-rh Just one suggestion from me. Otherwise, looks good! :fireworks:  When the change is made, we'll also just have to make sure that the multiple commits (currently 2) are squashed down into one.

**Comment by vikram-redhat** [comment]
@jhoyt-rh before merging we will need to make sure to remove the deleted file aggregate_logging.adoc from this commit. The deletion of that file shouldn't be part of this commit.

**Comment by jhoyt-rh** [comment]
Closing this PR due to the need to correct errors in file update creation process which caused the build to fail. Will open a new PR for this bug.

**Pull request closed without merging**